starknet_py/net/client_utils.py

```diff
@@ -79,10 +77,3 @@ def _is_valid_eth_address(address: str) -> bool:
     A function checking if an address matches Ethereum address regex. Note that it doesn't validate any checksums etc.
     """
     return bool(re.fullmatch("^0x[a-fA-F0-9]{40}$", address))
-
-
-def _create_broadcasted_txn(transaction: AccountTransaction) -> dict:
-    return cast(
-        Dict,
-        BroadcastedTransactionSchema().dump(obj=transaction),
-    )
```

Comment:
Why is this moved?

Reply:
1) It resolved cyclic dependency issue.
2) I decided that moving it closer to usage makes more sense than putting it in some generic utils file. This method is only extracted to be used in tests.

Reply:
Ok, sure